inference-engine/src/gna_plugin/transformations/decompose_2d_convolution.cpp

```diff
@@ -349,7 +370,7 @@ static std::shared_ptr<ngraph::Node> CreateDecomposedConv(const GraphData& graph
             // We need to calculate some parameters in case horizontal stride > 1 is used, because if we use the ones available from the original convolution
             // we won't take into account the fact horizontal strides will be supported by the newly created 1D convolution, and not by decomposition
             size_t filter_dilation_width = conv_data.filter_width > 1 ? conv_data.filter_dilation_width : 1;
-            size_t output_width = (conv_data.input_width - (conv_data.filter_width + filter_dilation_width - 2));
+            size_t output_width = (conv_data.input_width - (filter_dilation_width * (conv_data.filter_width - 1)));
```

Comment:
Please clarify two things:
1. Is `filter_dilation_width != 1` really supported here?
2. Is only convolution horizontal stride == 1 supported ?

Reply:
1. Yes, there are such cases in tests. Why?
2. No, please see comments. above the code. There are test cases with horizontal stride > 1. Again, why?

Reply:
Thanks for explanations.